Utilities/dronekit_wrappers.py

```diff
@@ -0,0 +1,93 @@
+# Standard Library
+import coloredlogs
+import dronekit
+import logging
+from math import radians, sin, cos
+from pymavlink import mavutil
+import threading
+from sys import stdout
+from time import sleep, time
+
+# Ours
+import constants as c
+
+def get_velocity_message(message_factory, (velocity_x, velocity_y, velocity_z)):
+    """
+    Returns a DroneKit object that represents a mavlink message which moves
+    the drone at a certain velocity.
+
+    Parameters
+    ----------
+    message_factory: dronekit.vehicle.message_factory
+        Used to create the message
+    (velocity_x, velocity_y, velocity_z): (Double, Double ,Double)
+        The vector to travel along
+
+    Returns:
+    ----------
+    MAVLink_message (a DroneKit object)
+    """
+    return message_factory.set_position_target_global_int_encode(
+            0,       # time_boot_ms (not used)
+            0, 0,    # target system, target component
+            mavutil.mavlink.MAV_FRAME_GLOBAL_RELATIVE_ALT_INT, # frame
+            0b0000111111000111, # type_mask (only speeds enabled)
+            0, # lat_int - X Position in WGS84 frame in 1e7 * meters
+            0, # lon_int - Y Position in WGS84 frame in 1e7 * meters
+            0, # alt - Altitude in meters in AMSL altitude(not WGS84 if absolute or relative)
+            # altitude above terrain if GLOBAL_TERRAIN_ALT_INT
+            velocity_x, # X velocity in NED frame in m/s
+            velocity_y, # Y velocity in NED frame in m/s
+            velocity_z, # Z velocity in NED frame in m/s
+            0, 0, 0, # afx, afy, afz acceleration (not supported yet, ignored in GCS_Mavlink)
+            0, 0)    # yaw, yaw_rate (not supported yet, ignored in GCS_Mavlink)
+   
+
+def set_attitude(vehicle, roll_angle = 0.0, pitch_angle = 0.0, yaw_rate = 0.0, thrust = 0.5):
+    """
+    NOTE: CURRENTLY NOT USED
+    The follow comments are from the DroneKit website:
```

Comment:
```suggestion
    The following comments are from the DroneKit website:
```

Reply:
Also you should probably link the URL as well instead of simply stating that they're from the website

Reply:
It seems I have erroneously put the comment under set_attitude() when it should be with send_global velocity(). I have changed its position and provided a link to the relevant web page.